components/expenses/ExpenseFormPayeeInviteNewStep.js

```diff
@@ -0,0 +1,322 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FastField, Field } from 'formik';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+
+import { ERROR, isErrorType } from '../../lib/errors';
+import { formatFormErrorMessage } from '../../lib/form-utils';
+
+import { Box, Flex, Grid } from '../Grid';
+import I18nAddressFields from '../I18nAddressFields';
+import { getI18nLink } from '../I18nFormatters';
+import InputTypeCountry from '../InputTypeCountry';
+import MessageBox from '../MessageBox';
+import StyledButton from '../StyledButton';
+import StyledHr from '../StyledHr';
+import StyledInput from '../StyledInput';
+import StyledInputField from '../StyledInputField';
+import StyledTextarea from '../StyledTextarea';
+
+import PayoutMethodForm from './PayoutMethodForm';
+import PayoutMethodSelect from './PayoutMethodSelect';
+
+const msg = defineMessages({
+  nameLabel: {
+    id: `ExpenseForm.inviteeLabel`,
+    defaultMessage: 'Who will receive the money for this expense?',
+  },
+  emailTitle: {
+    id: 'User.EmailAddress',
+    defaultMessage: 'Email address',
+  },
+  inviteeType: {
+    id: 'ExpenseForm.inviteeIsOrganizationLabel',
+    defaultMessage: 'Are you submitting this expense for your organization (company)?',
+  },
+  orgNameLabel: {
+    id: 'ExpenseForm.inviteeOrgNameLabel',
+    defaultMessage: "What's the name of the organization?",
+  },
+  orgSlugLabel: {
+    id: 'createCollective.form.slugLabel',
+    defaultMessage: 'Set your URL',
+  },
+  orgWebsiteLabel: {
+    id: 'createOrg.form.webstiteLabel',
+    defaultMessage: 'Organization website',
+  },
+  orgDescriptionLabel: {
+    id: 'ExpenseForm.inviteeOrgDescriptionLabel',
+    defaultMessage: 'What does your organization do?',
+  },
+  payoutOptionLabel: {
+    id: `ExpenseForm.PayoutOptionLabel`,
+    defaultMessage: 'Payout method',
+  },
+  invoiceInfo: {
+    id: 'ExpenseForm.InvoiceInfo',
+    defaultMessage: 'Additional invoice information',
+  },
+  invoiceInfoPlaceholder: {
+    id: 'ExpenseForm.InvoiceInfoPlaceholder',
+    defaultMessage: 'Tax ID, VAT number, etc. This information will be printed on your invoice.',
+  },
+  country: {
+    id: 'ExpenseForm.ChooseCountry',
+    defaultMessage: 'Choose country',
+  },
+  address: {
+    id: 'ExpenseForm.AddressLabel',
+    defaultMessage: 'Physical address',
+  },
+  recipientNoteLabel: {
+    id: 'ExpenseForm.RecipientNoteLabel',
+    defaultMessage: 'Add a note for the recipient',
+  },
+  additionalInfo: {
+    id: 'ExpenseForm.inviteAdditionalInfo',
+    defaultMessage:
+      'If you have additional information about the payee’s payment details, you can add them by clicking bellow.',
```

Comment:
@kewitz : Minor spelling and grammatical problems here I think; 1) the word below is spelled incorrectly, 2) I think `payee's payment details` is redundant; it could be shortened to just `payment details.

```suggestion
      'If you have additional information about the payment details, you can add them by clicking below.',
```